client/src/components/profile/components/TimeLine.js

```diff
@@ -42,7 +41,7 @@ const propTypes = {
     })
   ),
   fetchIdToNameMap: PropTypes.func.isRequired,
-  idToNameMap: PropTypes.objectOf(PropTypes.string),
+  idToNameMap: PropTypes.object,
```

Comment:
```suggestion
  idToNameMap: PropTypes.objectOf(
    PropTypes.shape({
      challengePath: PropTypes.string,
      challengeTitle: PropTypes.string
    })
  ),
```
Might as well specify the type of object.